cmd/ddev/cmd/debug-dockercheck.go

```diff
@@ -66,7 +66,10 @@ var DebugDockercheckCmd = &cobra.Command{
 				util.Warning("Problem with your Docker provider: %v.", err)
 			}
 		}
-
+		err = dockerutil.CheckDockerProvider()
+		if err != nil {
+			util.Warning("Problem with your Docker provider: %v.", err)
```

Comment:
To match the message in the `root.go`:

```suggestion
			util.Warning("Possible problem with your Docker provider: %v.", err)
```